src/chalk_common.nim

```diff
@@ -330,6 +330,8 @@ const
   baseConfName*       = "configs/base_*.c4m"
   sbomConfName*       = "configs/sbomconfig.c4m"
   sastConfName*       = "configs/sastconfig.c4m"
+  techStackConfName*  = "configs/techstackconfig.c4m"
+  linguistConfName*  = "configs/linguist.c4m"
```

Comment:
these would need to be added to:

* self extract which handles `load` command otherwise loading other components might be break if theyll reference anything defined in these components https://github.com/crashappsec/chalk/blob/f1ff5ded510171c9d4e025d06157929aafce1a4d/src/selfextract.nim#L217-L228
* self extract `nocache` list. otherwise these components will be cached into chalk mark during load is not expected https://github.com/crashappsec/chalk/blob/f1ff5ded510171c9d4e025d06157929aafce1a4d/src/selfextract.nim#L278-L286

note links are from my configs branch as changes were made there as well

Reply:
didnt see it before but I think you already did that